src/PKSim.UI/Views/Simulations/PopulationPKAnalysisView.cs

```diff
@@ -63,8 +80,8 @@ public override void InitializeResources()
          base.InitializeResources();
          btnExportToExcel.InitWithImage(ApplicationIcons.Excel, text: PKSimConstants.UI.ExportPKAnalysesToExcel);
          layoutItemExportToExcel.AdjustLargeButtonSize();
-         populationPKAnalysisItem.TextVisible = false;
          layoutControlItemGlobalPKAnalysis.TextVisible = false;
+         populationPKAnalysisXtraTabControl.SelectedPageChanged += (o, e) => OnEvent(_presenter.HandleTabChanged);
```

Comment:
Can we had icons to the tabs? One using the population icon and one using the timeProfile icon This would look a bit nicer. It might be enough to have the underling view return their corresponding icons depending on how the tabs are built